src/main/java/io/github/thebusybiscuit/slimefun4/core/SlimefunRegistry.java

```diff
@@ -200,7 +201,9 @@ public List<SlimefunItem> getEnabledSlimefunItems() {
      */
     @Nonnull
     public List<Research> getResearches() {
```

Comment:
When requesting a Set of all researches, one would expect to get the whole list, regardless of the researches current state.

If you want a list of enabled researches only, then a new method to return this should be made.

Reply:
Added a new method for this, should have done it this way to begin with - also fixed other usages and added specific placeholder for total researches enabled as `researches_total_researches_enabled`